Build failed; the context from the [latest run](https://ci.jenkins.io/job/Infra/job/evergreen/job/PR-25/1/display/redirect) is:
<details><summary>Expand to view</summary>

```
0 examples, 0 failures

Randomized with seed 34286

make: Leaving directory '/home/jenkins/workspace/Infra_evergreen_PR-25-JBRWILXK47P73DFZXTP3FANCAI77TAXXSYZ56UVAD76JSV76M7HQ/services'
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Build jenkins/evergreen)
[Pipeline] sh
[Infra_evergreen_PR-25-JBRWILXK47P73DFZXTP3FANCAI77TAXXSYZ56UVAD76JSV76M7HQ] Running shell script
+ make container
mkdir -p build
curl -sSL https://raw.githubusercontent.com/jenkinsci/docker/master//jenkins-support > build/jenkins-support
chmod +x build/jenkins-support
mkdir -p build
curl -sSL https://raw.githubusercontent.com/jenkinsci/docker/master//jenkins.sh > build/jenkins.sh
chmod +x build/jenkins.sh
mkdir -p build
curl -sSL https://raw.githubusercontent.com/jenkinsci/docker/master//install-plugins.sh > build/install-plugins.sh
chmod +x build/install-plugins.sh
git clone https://github.com/jenkinsci/configuration-as-code-plugin.git build/configuration-as-code
Cloning into 'build/configuration-as-code'...
cd build/configuration-as-code && mvn clean package -DskipTests
/bin/sh: 1: mvn: not found
Makefile:60: recipe for target 'build/configuration-as-code/target/configuration-as-code.hpi' failed
make: *** [build/configuration-as-code/target/configuration-as-code.hpi] Error 127
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Test jenkins/evergreen)
Stage 'Test jenkins/evergreen' skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] stage
[Pipeline] { (Publish jenkins/evergreen)
Stage 'Publish jenkins/evergreen' skipped due to earlier failure(s)
[Pipeline] }
[Pipeline] // stage
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline

GitHub has been notified of this commit’s build result

ERROR: script returned exit code 2
Finished: FAILURE

```
</details>

[Powered by the Comment Logger](https://github.com/jenkins-infra/community-functions/tree/master/comment-logger)

--- a/Dockerfile
+++ b/Dockerfile
@@ -77,6 +77,7 @@ COPY essentials.yaml /evergreen
 
 # FIXME (?): what if the end users touches the config value?
 # as is, we'll override it.
+COPY build/configuration-as-code/target/configuration-as-code.hpi /usr/share/jenkins/ref/plugins/configuration-as-code.hpi
 COPY jenkins-configuration.yaml /usr/share/jenkins/ref/jenkins.yaml
 ENV CASC_JENKINS_CONFIG=$JENKINS_HOME/jenkins.yaml
 
@@ -84,5 +85,3 @@ ENV CASC_JENKINS_CONFIG=$JENKINS_HOME/jenkins.yaml
 # that the Jenkins and evergreen-client processes both execute properly
 COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 CMD /usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf
-
-COPY configuration-as-code.hpi /usr/share/jenkins/ref/plugins/configuration-as-code.hpi

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -1,5 +1,9 @@
 pipeline {
     agent { label 'linux' }
+    tools {
+      maven 'mvn'
+      jdk 'jdk8'
+    }
 
     stages {
         stage('Prepare Workspace') {
@@ -28,7 +32,7 @@ pipeline {
 
         stage('Build jenkins/evergreen') {
             steps {
-                sh 'make container'
+              sh 'make container'
             }
         }
 

--- a/tests/tests.sh
+++ b/tests/tests.sh
@@ -16,7 +16,7 @@ oneTimeTearDown() {
 }
 
 test_smoke() {
-  curl --silent "http://localhost:$TEST_PORT > /dev/null"
+  curl --silent "http://localhost:$TEST_PORT" > /dev/null
   assertEquals "Bad exit" 0 $?
 }
 

